src/system_info/system_info.rs

```diff
@@ -0,0 +1,48 @@
+use rustc_serialize::{Decoder, Decodable};
```

Comment:
Rather than create a new module for this, can it be moved to the `config/system_info.rs` instead?